src/Spectre.Data/RoiIo/RoiUtilities.cs

```diff
@@ -0,0 +1,139 @@
+﻿/*
+ * RoiUtilities.cs
+ * Class with utilities for managing the regions of interest data.
+
+   Copyright 2017 Roman Lisak
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+namespace Spectre.Data.RoiIo
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Drawing;
+    using System.Drawing.Imaging;
+    using System.IO;
+    using System.Linq;
+    using Spectre.Data.Datasets;
+
+    /// <summary>
+    /// Class with utilities for managing the regions of interest data.
+    /// </summary>
+    /// <seealso cref="Spectre.Data.RoiIo.IRoiUtilites" />
+    public class RoiUtilities : IRoiUtilites
+    {
+        /// <summary>
+        /// The path
+        /// </summary>
+        private readonly string _path;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RoiUtilities"/> class.
+        /// </summary>
+        public RoiUtilities()
+        {
+            _path = System.IO.Path.GetFullPath(@"..\..\..\");
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RoiUtilities" /> class.
+        /// Used to pass exact path to methods. Default constructor sets project directory
+        /// as current path.
+        /// </summary>
+        /// <param name="testpath">The testpath.</param>
+        public RoiUtilities(string testpath)
+        {
+            _path = testpath;
+        }
+
+        /// <summary>
+        /// Reads regions of interest from a png file.
+        /// </summary>
+        /// <returns>
+        /// Returns list doubles.
+        /// </returns>
+        public RoiDataset RoiReader()
+        {
+            var clr = default(Color);
+            var bitmap = new Bitmap(_path);
+            var roidataset = new RoiDataset
+            {
+                RoiPixels = new List<RoiPixel>(),
+                Name = Path.GetFileNameWithoutExtension(_path),
+                Height = bitmap.Height,
+                Width = bitmap.Width
+            };
+
+            for (int width = 0; width < bitmap.Width; width++)
```

Comment:
Can we provide conversion to and from `Bitmap` separately to the readers and writers? It would be useful to have them separated - I see several use cases when it will be handy.